py/confluent/build/download_artifacts/nexusURL.py

```diff
@@ -0,0 +1,48 @@
+import os
+import re
+
+import requests
+
+from constants import BASE_URL, M2, PACKAGE_LITERAL, ARTIFACTID, USERNAME_REGEX, PASSWORD_REGEX, FILENAME_REGEX
```

Comment:
Use enums.
Follow import order.

Refer to import order here for example:
https://github.com/confluentinc/release-tools/blob/master/confluent/release/projects_editor.py